av/logging.pyx

```diff
@@ -106,7 +106,7 @@ def set_level(int level):
 
     Sets logging threshold when converting from FFmpeg's logging system
     to Python's. It is recommended to use the constants availible in this
```

Comment:
While we are fixing this section could we correct "availible" => "available" please?